### DISCUSSION TITLE
G Helper for Desktop Motherboards

### DISCUSSION THREAD

**Discussion by evenaardez** [seerge/g-helper#2591] [Ideas]
Hello, would it be possible to update G Helper to also work with motherboards such as the x670e extreme? This would be incredible as armoury crate is the only way to control the Anime Matrix as well as the LED screen and that is the single worst program I have ever used.

**Comment by seerge** [comment]
@evenaardez hello, this is a laptop app as it states in a title :)
Armoury for desktops is (de facto) absolutely different software (even tho under the same name)

P.S. If the only thing that you need is RGB controls, I would advice checking OpenRGB or SignalRGB projects
